x-pack/plugins/cases/server/services/attachments/index.ts

```diff
@@ -498,11 +494,11 @@ export class AttachmentService {
     return (
       res.aggregations?.references.caseIds.buckets.reduce((acc, idBucket) => {
         acc.set(idBucket.key, {
-          nonAlerts: idBucket.reverse.comments.doc_count,
```

Comment:
`nonAlerts` isn't really accurate anymore since the aggregation is filtering on type `user` so I renamed it.